src/tracing.rs

```diff
@@ -57,6 +57,34 @@ impl SpanProcessor for MetricsSpanProcessor {
             ])
             .collect::<Vec<_>>();
 
```

Comment:
Should this only be incremented on success code?

Reply:
what additional information does this metric provide that you can't already get from `rollup_boost_get_payload_v3_duration_count` filtering for payload_source and status = success?

Reply:
I guess it is just a rollup of the two types, so I will just sum over those two, thanks!